code/game/objects/items/food/_food.dm

```diff
@@ -26,14 +26,16 @@
 	var/list/eatverbs
 	///How much reagents per bite
 	var/bite_consumption
-	///What you get if you microwave the food, this should be replaced once I fully re-work cooking.
+	///What you get if you microwave the food. Use baking for raw things, use microwaving for already cooked things
 	var/microwaved_type
 	///Type of atom thats spawned after eating this item
 	var/trash_type
 	///How much junkiness this food has? God I should remove junkiness soon
 	var/junkiness
 	///Will this food turn into badrecipe on a grill? Don't use this for everything; preferably mostly for food that is made on a grill to begin with so it burns after some time
 	var/burns_on_grill = FALSE
+	///Will this food turn into badrecipe in an oven? Don't use this for everything; preferably mostly for food that is made in an oven to begin with so it burns after some time
```

Comment:
If that's the purpose wouldn't it be better for this to check if it's bakeable with positive result then add the negative one automatically ?